scripts/functions.ahk

```diff
@@ -272,9 +272,7 @@ SwitchInstance(idx, special:=False)
     SetAffinities(idx)
     if unpauseOnSwitch
```

Comment:
we should try

    if unpauseOnSwitch
      ControlSend,, {Blind}{Esc}, ahk_pid %pid%
    else
      ControlSend,, {Blind}{Esc 2}, ahk_pid %pid%

to try and get the GUI pause menu instead of f3+esc pause menu as ssg players want to press the 'back to game' button, not esc